source/deployment/sso-saml-adfs.md

```diff
@@ -0,0 +1,213 @@
+
+![adfs_19_export_idp_cert_start.PNG](../../source/images/adfs_19_export_idp_cert_start.PNG)
+
+25) In the **Certificate** screen, go to the **Details** tab and click **Copy to File…**, then **OK**. This opens a **Certificate Export Wizard**.
+
+![adfs_20_export_idp_cert_copy.PNG](../../source/images/adfs_20_export_idp_cert_copy.PNG)
+
+26) In the **Certificate Export Wizard** screen, click **Next**. Then, select the option **Base-64 encoded X.509 (.CER)** and click **Next** again.
+
+![adfs_21_export_idp_cert_wizard.PNG](../../source/images/adfs_21_export_idp_cert_wizard.PNG)
+
+27) In the **Certificate Export Wizard** screen, click **Browser** to specify the location you want the Identity Provider Certificate to be exported, and specify the file name.
+
+![adfs_21-2_export_idp_cert_wizard.PNG](../../source/images/adfs_21-2_export_idp_cert_wizard.PNG)
+
+28) Click **Save**. In the **Certificate Export Wizard** screen, verify the file path is correct, and click **Next**. 
+
+29) In the **Completing the Certificate Export Wizard**, click **Finish**, then **OK** to confirm the export was successful.
+
+![adfs_21-3_export_idp_cert_wizard.PNG](../../source/images/adfs_21-3_export_idp_cert_wizard.PNG)
+
+You’re now about to finish configuring SAML for Mattermost!
+
+### Configure SAML for Mattermost
+
+30) Start Mattermost server and sign into Mattermost as a System Administrator. Go to **System Console > Authentication > SAML**.
+  - `SAML SSO URL`: **SAML 2.0/W-Federation URL** ADFS Endpoint you noted in step 1.
+  - `Identity Provider Issuer URL`: `Relying party trust identifier` from ADFS you specified in step 11.
+  - `Identity Provider Public Certificate`: X.509 Public Certificate you downloaded in step 29.
+
+![adfs_22_mattermost_basics.PNG](../../source/images/adfs_22_mattermost_basics.PNG)
+
+31) (Optional) Configure Mattermost to verify the signature. The `Service Provider Login URL` is the `SAML 2.0 SSO service URL` you specified in ADFS in step 10.
+
+![adfs_23_mattermost_verification.PNG](../../source/images/adfs_23_mattermost_verification.PNG)
+
+32) (Optional) Enable encryption by uploading the Service Provider Private Key and Service Provider Public Certificate you generated  in step 3.
+
+![adfs_24_mattermost_encryption.PNG](../../source/images/adfs_24_mattermost_encryption.PNG)
+
+33) Set attributes for the SAML Assertions, which will be used to update user information in Mattermost. Attributes for email, username, first name and last name are required and should match the values you entered in ADFS in step 15. See [documentation on SAML configuration settings](http://docs.mattermost.com/administration/config-settings.html#saml-enterprise) for more detail.
+
+![adfs_25_mattermost_attributes.PNG](../../source/images/adfs_25_mattermost_attributes.PNG)
+
+34) (Optional) Lastly, customize the login button text.
+
+![adfs_26_mattermost_login_button.PNG](../../source/images/adfs_26_mattermost_login_button.PNG)
```

Comment:
I think Lindsay is correct, it should say "With ADFS" for the ADFS docs

I'm 0/5 on whether `Enterprise SSO` is better, but I've seen `With Okta` and `With ADFS` used elsewhere, so I'll change it to that for now